vulnfeeds/cmd/nvd-cve-osv/run_cve_to_osv_generation.sh

```diff
@@ -56,6 +60,14 @@ for (( YEAR = $(date +%Y) ; YEAR >= ${FIRST_INSCOPE_YEAR} ; YEAR-- )); do
 done
 
 # Copy results to GCS bucket.
+objs_present=$(gsutil ls "${OSV_OUTPUT_GCS_PATH}" | wc -l)
+objs_deleted=$(gsutil -m rsync -n -c -d "${WORK_DIR}/nvd2osv/gcs_stage" "${OSV_OUTPUT_GCS_PATH}" 2>&1 | grep -c "Would delete")
+
+if (( objs_deleted < objs_present * 0.1 )); then
```

Comment:
should this be > instead?

Reply:
Yes. Fixed this, and many other things surfaced from rigorous manual testing in [bb87c97](https://github.com/google/osv.dev/pull/2732/commits/bb87c979c8c58eba16e56c223f263076c542266c)